<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Frame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_bar">
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
